api/fittings/lisk_error_handler.js

```diff
@@ -26,17 +24,23 @@ module.exports = function create (fittingDef, bagpipes) {
 			}
 		}
 
-		if (context.statusCode === 500 && !fittingDef.handle500Errors) {
-			return next(err);
+		debug('exec: %s', context.error.message);
+		debug('status: %s', context.statusCode);
+
+		if (context.statusCode === 500) {
+			if(!fittingDef.handle500Errors) {
+				return next(err);
+			}
+
+			err = {
+				message: 'An unexpected error seems to have occurred. You can try again or contact us if problem persist.'
```

Comment:
who is us?

Reply:
Lisk support team.

Reply:
If we don't have further context to add, I would only return something like `An unexpected error has occurred while handling this request`.

Reply:
We aren't the only people running nodes, so this is not applicable @nazarhussain 

The response @karmacoma has here is what we should use